src/include/duckdb/main/query_profiler.hpp

```diff
@@ -114,6 +119,10 @@ class OperatorProfiler {
 
 	//! Whether or not the profiler is enabled
 	bool enabled;
+	//! Where or not the timing profiler is enabled
+	bool operator_timing_enabled = false;
+	//! Whether or not the cardinality profiler is enabled
+	bool operator_cardinality_enabled = false;
```

Comment:
How can we remove these booleans, as they're already implicitly stored via the map? Maybe we should keep the map in each operator's profiler? As that's the struct that we'll use in the different operators 🤔

Reply:
Yes I like this idea, unless there are things that we keep track of that aren't calculated in the operator? Is that possible?